python/lsst/meas/extensions/photometryKron/kronLib.i

```diff
@@ -37,15 +37,39 @@ Interface to Kron magnitudes
 #include "lsst/afw/detection/Footprint.h"
 #include "lsst/afw/image/Utils.h"
 #include "lsst/afw/geom/Point.h"
-#include "lsst/meas/algorithms.h"
-#include "lsst/meas/extensions/photometryKron.h"
+#include "lsst/afw/math.h"
+#include "lsst/afw/image.h"
+#include "lsst/afw/detection.h"
+#include "lsst/afw/table.h"
+#include "lsst/meas/base.h"
+#define PY_ARRAY_UNIQUE_SYMBOL LSST_MEAS_BASE_NUMPY_ARRAY_API
```

Comment:
A line like this is needed any time you want to use one of `ndarray`s converters to/from `NumPy` arrays, including those for Eigen objects.  It looks like you're declaring at least one such converter below (the call to `%declareNumPyConverters(...)`.  My guess is that you don't actually need that here, since I don't think the Kron code ever measures a centroid, and hence you could remove this line and all the others that include ndarray or NumPy headers, and the `%init` block that initializes the NumPy library.